### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by pranavbharadwaj007**
Simple footer

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+                </div>
+                
+                <div className="socials">
+                  <a href="https://www.linkedin.com/company/is-technical-club-sce"><FontAwesomeIcon
+                                style={fontAwesomeStyle}
+                                icon={faLinkedin}
+                                size="2x"
+                              /></a>
+                              <a href="https://www.instagram.com/istcsce/?igshid=auqnvolgbjii">
```
Comment (istc-sce) [inline src/components/footer.js:33]: Open the links in new tab. We dont want the user to leave our site.

File: src/components/footer.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                              <FontAwesomeIcon
+                                style={fontAwesomeStyle}
+                                icon={faInstagram}
+                                size="2x"
+                              />
+                              </a>
+               <a href="https://github.com/niranjanhegde144/ISETechClub">
```
Comment (istc-sce) [inline src/components/footer.js:40]: Change repo link, and open in new tab

File: src/components/footer.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            <div className="footer">
+               <div className="footer-content">
+                <div className="footer-section about">
+                <h1 className="logo-text"> Follow US</h1>
+                <div className="follow">
+
+                </div>
+                
+                <div className="socials">
+                  <a href="https://www.linkedin.com/company/is-technical-club-sce"><FontAwesomeIcon
```
Comment (istc-sce) [inline src/components/footer.js:28]: Open in new tab.

File: src/components/footer.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                 </li>
+                 <li>
+            <Link as={Link} to="/events">
+              Events
+            </Link>
+                 </li>
+                 <li>
+            <Link  as={Link} to="/programming">
+              Programming
+            </Link>
```
Comment (istc-sce) [inline src/components/footer.js:66]: We are working on something, so we wont have this /programming link available.

So for now you can just comment this part, and edit it post those changes.

File: src/components/footer.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+                    <h1>
+                        Contact Us
+                    </h1>
+                    <div className="contactwe">
+                   
+                                <h3> <FontAwesomeIcon
+                                style={fontAwesomeStyle}
+                                icon={faEnvelope}
+                                size="sm"
+                                /> istcsce@gmail.com</h3>
```
Comment (istc-sce) [inline src/components/footer.js:85]: use "mailto:" here.

File: src/components/footer.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+          <p>&#169; copyright reserved || ISTC </p> 
+               </div>
+            </div>
+            
+        )
+    }
+}
+
+
+export default Footer;
```
Comment (istc-sce) [inline src/components/footer.js:99]: Check the mobile view of footer as well. Need to do some fixes for that.

**Review by istc-sce — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 7]
```diff
@@ -1,344 +1,405 @@
...
-  color: rgba(18, 26, 32, 0.952) !important;
+    color: rgba(18, 26, 32, 0.952) !important;
 }
 
 * {
-  font-family: "Montserrat", sans-serif;
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+    font-family: "Montserrat", sans-serif;
```
Comment (niranjanhegde144) [inline src/App.css:18]: Font family is not required remove it.

File: src/App.css [hunk 8]
```diff
@@ -1,344 +1,405 @@
...
-  font-family: "Montserrat", sans-serif;
-  font-size: 50px;
+    font-family: "Montserrat", sans-serif;
+    font-size: 50px;
 }
 
 .text-medium {
-  font-family: "Montserrat", sans-serif;
-  font-size: 35px;
+    font-family: "Montserrat", sans-serif;
```
Comment (niranjanhegde144) [inline src/App.css:136]: Remove font family everywhere.

File: src/components/footer.js (hunk 5 above)
Comment (akramkazmi71) [reply src/components/footer.js:85]: You missed this comment, I guess.

File: src/components/footer.js [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+                    <h1>
+                        Contact Us
+                    </h1>
+                    <div className="contactwe">
+                   
+                                <h3> <FontAwesomeIcon
+                                style={fontAwesomeStyle}
+                                icon={faEnvelope}
+                                size="sm"
+                                /> mailto: istcsce@gmail.com</h3>
```
Comment (akramkazmi71) [inline src/components/footer.js:85]: 'mailto' is referred to this:
`<a href = "mailto: istcsce@gmail.com">istcsce@gmail.com</a>`

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Pull request merged**